core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1119,6 +1119,16 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
     logProps
   }
 
+  /**
+   * Copy the subset of properties that are relevant to consumer group.
+   */
+  def extractGroupConfigMap: java.util.Map[String, Int] = {
+    val groupProps = new java.util.HashMap[String, Int]()
+    groupProps.put(GroupConfig.CONSUMER_SESSION_TIMEOUT_MS_CONFIG, groupCoordinatorConfig.consumerGroupSessionTimeoutMs)
+    groupProps.put(GroupConfig.CONSUMER_HEARTBEAT_INTERVAL_MS_CONFIG, groupCoordinatorConfig.consumerGroupHeartbeatIntervalMs)
+    groupProps
+  }
```

Comment:
Would it make sense to move this method to `GroupCoordinatorConfig`? `GroupCoordinatorConfig` has changed quite a lot since this PR was started. It seems to me that keeping all the group related logic there would make sense. What do you think?

If we do this, we could also simplify `GroupConfigManager.validate`.

Reply:
Yes it makes sense to me. Moved this method to `GroupCoordinatorConfig`.